data/nlu.md

```diff
@@ -84,6 +84,11 @@
 - are [pichuka](pokemon_name) and [charmander](pokemon_name) pokemon?
 - are [snorlax](pokemon_name), [gariddos](pokemon_name) and [dittto](pokemon_name) pokemon?
 
+## intent:name_translation
+- What is the [french](language) name of [Snorlax](pokemon_name)?
+- What is the [spanish](language) name of [Snorlax](pokemon_name)?
+- What is the [german](language) name of [Snorlax](pokemon_name)?
+
```

Comment:
It may be good to also add some variants of language here, otherwise the chatbot is at risk of literally learning this sentence structure. Might be good to add some examples like; 

```
- Can you translate [Snorlax](pokemon_name) into [french](language)
- Translate [Snorlax](pokemon_name) to [french](language)
- How would you say [Snorlax](pokemon_name) in [french](language)
```

Reply:
A small oversight from my part. I'll change it :)

Reply:
Don't worry about it being perfect though, we can always add more examples and folks can make PRs for this.